### ISSUE TITLE
Как сам, братишка?

### ISSUE THREAD

**Issue by matterai** [CassandraDied/CPUStress#1]

**Comment by vldalx** [comment]
У меня та же херня с эти кодом!!! Автора давай фикси! Я использую этот код на омной высоконагруженном проекте!!!!!!!!!!

**Comment by CassandraDied** [comment]
Нашли, куда написать, лол.
Чёт я пригрустил. Как-то всё лениво и Рихтер уже не торт. Три дня разбирался, как IDispossable должен работать и информации столько, что можно писать статью.
Сижу в квартирке, Хауса смотрю. Пописываю время от времени утилитки. Хром что-то грузит так сильно, что звук прерывается. Надо было проверить, что это не CPU.
Говорят, у вас там куча людей теперь на проекте, ажно 4 заднего конца разработчика. Справляетесь?
@matterai @vldalx

**Comment by vldalx** [comment]
@CassandraDied настало время поведать миру про [IDispossable](https://msdn.microsoft.com/en-us/library/system.idisposable(v=vs.110).aspx) на [dotnext](https://dotnext-piter.ru/)?

**Comment by CassandraDied** [comment]
@vldalx вряд ли. На Хабр потянет, а так — скукатень. Настало время начать искать работу. Чятик ещё не переписали? А то давно пора.

**Comment by vldalx** [comment]
@CassandraDied спеца по бэку чатика мы потеряли =( некому переписывать
но @matterai не неделе в очередной раз его пофиксил.

**Comment by CassandraDied** [comment]
@vldalx офигеть, там и дефекты были? Не ожидал от тебя такого, Володя. Все же хорошо было, как так-то?
@matterai классно, что чятик починил. Я переживал, что там всё будет настолько через жопу, что любой фикс выведет что-то другой из рабочего состояния.

**Comment by vldalx** [comment]
дифектов не было на клиенте. я, кстати, либу новую вворотил для SignalR вместо старой Objectiv-C'шной.
чатик не работал только на одном из серверов и @matterai все по красоте сделал.
@CassandraDied не хочешь [так](https://vimeo.com/212425775) работать

**Comment by CassandraDied** [comment]
@vldalx чем она лучше? Сколько дней потом проверял, что всё работает? Не работал на одном из серверов или на одной из нод?
>EPAM
No, thanks. EPAM, VEEAM и прочие “мы не можем сказать, на каком проекте ты будешь работать” идут лесом. Алсо, компании с >200 человек. За исключением, если это какой-то одтельный проект, на котором уже всё известно и тебя не переместят на другой через пару месяцев. Алсо, эффективные менеджеры тоже идут лесом. Мне почти негде работать!

**Comment by vldalx** [comment]
@CassandraDied написана на чистом Swift. я там 6 пулл реквестов запилил и все как доктор прописал

**Comment by CassandraDied** [comment]
> написана на чистом Swift. я там 6 пулл реквестов запилил

Отвалится в самый неожиданный и неудобный момент. Не хочешь тестов запилить? Тебе наверняка всё равно заняться там нечем.

**Comment by vldalx** [comment]
@CassandraDied  старая не надежнее была
а еще нам потребовалось добавить поддержду самоподписанных SSL сертификатов и старая либа соснула, чат нахрен отвалился

**Comment by CassandraDied** [comment]
@vldalx а новая это таки умеет? Удивительно. Сертификаты на стороне клиента? Я думал, они только на сервере нужны. Или это не связано с чатом?

**Comment by vldalx** [comment]
новую научили с этим работать. так там просто проверка сертификата не проходит и все отваливатеся. вот нужно было отключить проверку сертификата

**Comment by CassandraDied** [comment]
Проверка по спискам доверенных authority?

**Comment by vldalx** [comment]
ну да, самоподписанный сертификат не считается настоящим. для тестов только надо

**Comment by vldalx** [comment]
@CassandraDied а еще у нас прожет мэнеджер будет в течение 2-х недель. вот теперь заживем

**Comment by CassandraDied** [comment]
@vldalx ну да, он точно напишет весь нужный код и починит дефекты. Работать будет за шестерых.

**Comment by vldalx** [comment]
@CassandraDied он воплотит все твои мечты о процессе разработки в жизнь

**Comment by matterai** [comment]
@CassandraDied говна на лопате, лучше пить водку и кушать картошечку, Артём не ходи на работу - Хаус и утилитки гораздо лучше, точно тебе говорю

**Comment by matterai** [comment]
@CassandraDied кстати в чатике ничего такого не было, все нормально работало, просто когда на azure devenv переносили в app.config не изменили `serviceBusName` со старого на новый, и от этого всё окосело, он просто Factory не мог создать, потому что Exception был. Пофиксил appconfig и все задышало. Т.ч. твой код робит отлично.

**Comment by CassandraDied** [comment]
@matterai 
> Артём не ходи на работу - Хаус и утилитки гораздо лучше, точно тебе говорю

Чёт у меня аллергия какая-то на это дерьмо. Я перестал есть, перестал спать. И делать ничего не хочется. Не, я так больше не могу. Без каждодневного дышания пылью в метро жизнь бессмысленна. А этот офисный кофе, ммм. Придётся всё-таки разместить резюме. Я несколько дней ни с кем голосом не говорил уже, ёпт.

**Comment by vldalx** [comment]
> Без каждодневного дышания пылью в метро жизнь бессмысленна.

т.е. работу в пешей доступности от дома ты не рассматриваешь?

**Comment by CassandraDied** [comment]
@vldalx нет, конечно. Я ещё люблю прохаживаться часа по два от офиса до квартиры время от времени.
Завтра буду в кофеенке рядом с офисом часов в 17. Можно постоять, покурить, если у кого будет желание. Расскажу, как зафейлил интевью в Лабораторию Касперского.

**Comment by CassandraDied** [comment]
@vldalx а, ты ж бросил. Всё забываю.

**Comment by vldalx** [comment]
@CassandraDied так домой можно идти не по прямой. можно полдня гулять до места, которое в 100 метрах от тебя.
как подойдешь, то пиши, завтра видно будет
